Cargo.toml

```diff
@@ -48,6 +48,7 @@ globset = "0.4"
 anyhow = "1.0"
 dirs-next = "2.0"
 normpath = "0.3"
+chrono = "0.4.19"
```

Comment:
Do we need to specify `.19` here or would `0.4` be okay? Explicit versions are set in `Cargo.lock` anyways.